library/src-bootstrapped/scala/quoted/QuoteContext.scala

```diff
@@ -5,17 +5,20 @@ import scala.quoted.show.SyntaxHighlight
 /** Quotation context provided by a macro expansion or in the scope of `scala.quoted.run`.
  *  Used to perform all operations on quoted `Expr` or `Type`.
  *
- *  It contains the low-level Typed AST API `tasty` metaprogramming API.
+ *  It contains the low-level Typed AST API metaprogramming API.
  *  This API does not have the static type guarantiees that `Expr` and `Type` provide.
  *
- *  @param tasty Typed AST API. Usage: `def f(qctx: QuoteContext) = { import qctx.tasty._; ... }`.
+ *  @param tasty Typed AST API. Usage: `def f(qctx: QuoteContext) = { import qctx.ast._; ... }`.
  */
 trait QuoteContext { self =>
 
-  /** Low-level Typed AST API `tasty` metaprogramming API.
+  /** Low-level Typed AST API metaprogramming API.
    *  This API does not have the static type guarantiees that `Expr` and `Type` provide.
    */
-  val tasty: scala.tasty.Reflection
+  val ast: scala.tasty.Reflection
```

Comment:
Why not just call it `reflect`, as it's not just about AST?

Reply:
That is a good alternative. It is even more meaningful. The only downside is length, which might not be an issue based on the current diff. I will try this alternative.